vital/types/image_container_set_simple.cxx

```diff
@@ -0,0 +1,122 @@
+/*ckwg +29
```

Comment:
Minor nit, but why is the class named differently from the cxx file. That, IMHO can get confusing when looking for code.

Reply:
The class-name is the same as it had been before, but I think I named the file such that the source file would lexicographically sort next to abstract base source file.  Are you suggesting that the source file or class name be changed (if you have a side)?

Reply:
The file should be named same as the code. Not sure what happened here.